.storybook/time-machine.js

```diff
@@ -4,10 +4,14 @@ timemachine is a node module which overrides the system time for a repo. There a
 */
 
 // Sets the date and time whilst leaving the clock ticking from that point onwards.
-export const startTimeMachine = (tick = true) => {
+export const startTimeMachine = ({
+  tick = true,
+  dateString = 'Tuesday, 1 January 2019 02:00:00 GMT',
+  timestamp = 1546308000,
+}) => {
```

Comment:
I've updated the default time for Time Machine to be the 1st January 2019 so that it is clear that this is an artificially-set date.

Reply:
@sareh This is more of a question, not a review: The previous date was very specific, I'm not sure why but is it possible there was a reason it was so specific? Probably based on some unique stories in Storybook?

Reply:
The other storybook stories that are using Timestamp are these:
```
Containers|Article/Article Timestamp
Containers|MAP Media Player/Canonical
Containers|MAP Media Player/AMP
Moment Locales/Editorial view
Moment Locales/Developer view
Pages|Front Page
Pages|Media Asset Page/Canonical
Pages|Media Asset Page/AMP
Pages|OnDemand Radio Page
Pages|Photo Gallery Page
Pages|Radio Page
Pages|Story Page
```
It would be good to get reviews of these stories from people in the WS Stories and WS Media teams to ensure that the specific date (Friday, 9 August 2019 14:04:14 GMT) is not needed for them. If this specific date _is_ required, I can explicitly add this date to those stories.

Reply:
The specific date has no impact on us - looks good to me 👍